src/main/java/com/exadel/discount/controller/CouponController.java

```diff
@@ -89,7 +87,7 @@ public CouponDTO getCouponById(@PathVariable @NotNull final UUID id) {
     @PostMapping
     @ApiOperation("Save new coupon to user")
     @UserAccess
-    public CouponDTO addCoupon(@RequestBody @NotNull final CreateCouponDTO createCouponDTO) {
```

Comment:
If you don't need CreateCouponDTO and CreateFavoriteDTO anymore, you can delete these files

Reply:
Done